singlevenue/src/main/res/layout/activity_main.xml

```diff
@@ -50,16 +50,17 @@
                     android:layout_height="match_parent"
                     android:layout_alignParentBottom="true"
                     android:layout_marginLeft="15dp"
-                    android:layout_marginRight="15dp"
                     android:layout_marginTop="15dp"
+                    android:layout_marginRight="15dp"
+                    android:layout_marginBottom="-3dp"
```

Comment:
I think this value should be adjusted so that the go button is in line with the level changer